docs/admin/auth.md

```diff
@@ -1,7 +1,5 @@
 # Authentication
 
-![OIDC with Coder Sequence Diagram](../images/oidc-sequence-diagram.svg).
-
```

Comment:
Do we have an issue to create a new diagram?

Reply:
No I just want to remove it for now